index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html ng-app>
+  <head>
```

Comment:
this is a practice comment, to show how to make comments when doing a code review. this is my PR, so i don't need to make a real comment.